### PR TITLE
Symfony update to 2.7.5

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q                    | A
| -------------       | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

Update due to the 'case mismatch' bug detected while setting up Sylius locally. 
PR from Symfony: https://github.com/symfony/symfony/pull/15783

### CODE REVIEW

**Comment by pamil** [comment]
We need to bump requirements in `composer.json` too.

**Comment by TheMadeleine** [comment]
@pamil  :+1:

File: composer.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
         "symfony/options-resolver":             "^2.7",
         "symfony/property-access":              "^2.7",
         "symfony/swiftmailer-bundle":           "^2.3",
-        "symfony/symfony":                      "^2.7",
+        "symfony/symfony":                      "^2.7.5",
```
Comment (michalmarcinkowski) [inline composer.json]: Proper criteria should be `>=2.7.5, <3.0.0`

Comment (pamil) [inline composer.json]: `^2.7.5` means the same :)

Comment (michalmarcinkowski) [inline composer.json]: I somehow saw tilde instead of caret :smile: my bad then :wink:

**Comment by pjedrzejewski** [comment]
@pamil Do you mean just main ``composer.json``? (singular form) If yes, then I will add that we should adjust all component&bundle jsons.

**Comment by TheMadeleine** [comment]
@pjedrzejewski I'll take care of it ASAP :)

**Comment by TheMadeleine** [comment]
Okay @pjedrzejewski what if, we don't have any dependencies to symfony/symfony or symfony/debug (which was being fixed by that PR in symfony)? :) Soo all the component & bundle jsons are already adjusted, am I right?

**Comment by pjedrzejewski** [comment]
Thank you Magda!

**Pull request merged**